bot/code_review_bot/report/base.py

```diff
@@ -4,34 +4,17 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import itertools
+import re
 import urllib.parse
-
-from code_review_bot.tasks.clang_format import ClangFormatIssue
-from code_review_bot.tasks.clang_tidy import ClangTidyIssue
-from code_review_bot.tasks.coverity import CoverityIssue
-from code_review_bot.tasks.default import DefaultIssue
-from code_review_bot.tasks.infer import InferIssue
-from code_review_bot.tasks.lint import MozLintIssue
-
-COMMENT_PARTS = {
-    ClangTidyIssue: {
-        "defect": " - {nb} found by clang-tidy",
-        "analyzer": " - `./mach static-analysis check {files}` (C/C++)",
-    },
-    InferIssue: {
-        "defect": " - {nb} found by infer",
-        "analyzer": " - `./mach static-analysis check-java path/to/file.java` (Java)",
-    },
-    CoverityIssue: {"defect": " - {nb} found by Coverity"},
-    ClangFormatIssue: {
-        "defect": " - {nb} found by clang-format",
-        "analyzer": " - `./mach clang-format -s -p {files}` (C/C++)",
-    },
-    MozLintIssue: {
-        "defect": " - {nb} found by mozlint",
-        "analyzer": " - `./mach lint --warnings path/to/file` (JS/Python/etc)",
-    },
-    DefaultIssue: {"defect": " - {nb} found by a generic analyzer"},
+from typing import Pattern
+
+HELP_COMMANDS = {
```

Comment:
Thought for the future: It'd be nice if the task defined the local command to use in its `extra`, so we don't have to hardcode it here.

Reply:
Totally agree !

Reply:
https://github.com/mozilla/code-review/issues/339